frontend/src/scenes/persons/personLogic.tsx

```diff
@@ -29,8 +38,50 @@ export const personLogic = kea<personLogicType>([
                 },
             },
         ],
+        info: [
+            null as Info | null,
+            {
+                loadInfo: async (): Promise<Info | null> => {
+                    if (!props.id) {
+                        return null
+                    }
+
+                    const infoQuery = hogql`
+                    SELECT
+                        count(DISTINCT $session_id) as session_count,
+                        count(*) as event_count,
+                        max(timestamp) as last_seen
+                    FROM events
+                    WHERE person_id = ${props.id}
+                    AND timestamp >= now() - interval 30 day
+                    `
+                    try {
+                        const response = await api.queryHogQL(infoQuery)
+                        const row = response.results?.[0]
+                        if (!row) {
+                            return {
+                                sessionCount: 0,
+                                eventCount: 0,
+                                lastSeen: null,
+                            }
+                        }
+
+                        const [sessionCount, eventCount, lastSeen] = row
+                        return { sessionCount, eventCount, lastSeen }
+                    } catch (error: any) {
+                        posthog.captureException(error)
+                        return {
+                            sessionCount: 0,
+                            eventCount: 0,
+                            lastSeen: null,
+                        }
+                    }
+                },
+            },
+        ],
     })),
     afterMount(({ actions }) => {
         actions.loadPerson()
+        actions.loadInfo()
     }),
```

Comment:
You could use `lazyLoaders` rather than `loaders` to automate this bit, and avoid loading it before we actually use the property